hubspot-client-bundles/hbase-mapreduce-bundle/pom.xml

```diff
@@ -208,6 +218,7 @@
                   to manage those versions in parent-pom
                   -->
                   <include>org.apache.hbase.thirdparty:*</include>
+                  <include>com.github.luben:zstd-jni</include>
```

Comment:
I don't think we want this. Instead we let it get pulled in transitively. That way we could manage this in parent-pom if we want. The goal is that the bundles are just hbase code. Otherwise if this bundle is pulled in to a module that also separately depeneded on zstd-jni, there'd be conflict classpath issues.